### ISSUE TITLE
Failure occurs when the Reynolds number increases

### ISSUE THREAD

**Issue by chenchenchen7** [okada39/pinn_cavity#1]
Dear Sir

I found that when the Reynolds number becomes larger (Re=1000, nu=0.001), the effect is very poor from the drawn image.
I don’t understand why this is happening. Could it be that the equation is no longer suitable?

thanks for replying

**Comment by okada39** [comment]
Dear chenchenchen7,

We verified that PINN algorithm cannot get an ideal convergence for complex phenomena such as high Reynolds flow and high-dimensional equation. Deep neural networks potentially represent their solutions, but localized ones constrict even though we use quasi-Netwon method. 

This problem is a challenge to overcome for implementing PINN application.

**Comment by AmirhosseinnnKhademi** [comment]
But the problem is that for cases such as Re=10000! the loss values remains the same as the case for Re=400!
Why is that? it does not make sense!

**Comment by AmirhosseinnnKhademi** [comment]
Also, it seems to me that you cannot define separate numbers of training and collocation points for the training! why is that?
